client/src/components/categorical/category.js

```diff
@@ -164,6 +170,11 @@ class Category extends React.Component {
       false
     );
 
+    const truncatedString = maybeTruncateString(
```

Comment:
why are value.js and category.js using different functions to truncate the string?

Reply:
I just learned about this function while working on this PR, I'll be utilizing this function when addressing #1209